helpdesk_mgmt/__manifest__.py

```diff
@@ -4,14 +4,15 @@
     'name': 'Helpdesk Management',
     'summary': """
         Helpdesk""",
-    'version': '11.0.1.1.0',
+    'version': '12.0.1.0.0',
     'license': 'AGPL-3',
     'category': 'After-Sales',
     'author': 'AdaptiveCity, '
               'C2i Change 2 Improve, '
               'Domatix, '
               'Factor Libre, '
               'SDi Soluciones, '
+              'ID42 Soluções, '
```

Comment:
You mustn't put as co-author for this little work

Reply:
I agree